tests/network/vmi_networking.go

```diff
@@ -72,12 +70,14 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 		LibvirtBlockMigrationPort  = 49153
 	)
 
-	tests.BeforeAll(func() {
+	BeforeEach(func() {
 		virtClient, err = kubecli.GetKubevirtClient()
 		util.PanicOnError(err)
 
 		kv := util.GetCurrentKv(virtClient)
 		currentConfiguration = kv.Spec.Configuration
```

Comment:
If you want to modify the configuration, you should take a snapshot after the cleanup took place.